test/test_functional_tensor.py

```diff
@@ -295,6 +211,88 @@ def test_scale_channel(self):
         assert_equal(scaled_cpu, scaled_cuda.to('cpu'))
 
 
+class TestRotate:
+
+    ALL_DTYPES = [None, torch.float32, torch.float64, torch.float16]
+    scripted_rotate = torch.jit.script(F.rotate)
+    IMG_W = 26
```

Comment:
I’m not sure if the widths of the two images are equal by coincidence, but under this condition, the `center` can be parametrized.